src/include/cpu/power/isteps.h

```diff
@@ -0,0 +1,58 @@
+#include <stdint.h>
```

Comment:
No SPDX header at the beginning of the file. Also I would rename this file to istep8.h we don't want one header file for all isteps

Reply:
I have added the header and renamed the file.